js/github.js

```diff
@@ -0,0 +1,27 @@
+// Pulls the last modified date of the current documentation from GitHub
+
+var fetch_github_date = function(callback) {
+    if (typeof github_repo == "undefined" || typeof github_path == "undefined")
```

Comment:
Nitpick: Can we avoid 1 line if statements. Either wrap the block or put the return on the same line. It's error prone if someone adds to this block without thinking it through.